panel/chat/feed.py

```diff
@@ -35,58 +35,6 @@
     from bokeh.model import Model
     from pyviz_comms import Comm
 
-USER_LOGO = "🧑"
```

Comment:
Was wondering why adding default avatars here didn't work; turns out this was migrated to `message.py` and this was left behind.
https://github.com/holoviz/panel/blob/main/panel/chat/message.py#L48-L101